protocol/v2/ssv/validator/non_committee_validator.go

```diff
@@ -278,8 +278,6 @@ func (ncv *CommitteeObserver) resolveDuplicateSignature(container *ssv.PartialSi
 
 // copied from BaseRunner
 func (ncv *CommitteeObserver) verifyBeaconPartialSignature(signer uint64, signature spectypes.Signature, root phase0.Root, share *ssvtypes.SSVShare) error {
-	ssvtypes.MetricsSignaturesVerifications.WithLabelValues().Inc()
```

Comment:
for future versions this metric might be interesting when looking into the performance of our node.